pom.xml

```diff
@@ -37,6 +37,9 @@
   <properties>
     <gitHubRepo>jenkinsci/dark-theme-plugin</gitHubRepo>
     <changelist>9999999-SNAPSHOT</changelist>
+    <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
+    <jenkins.baseline>2.426</jenkins.baseline>
+    <!-- TODO: should depend on baseline -->
     <jenkins.version>2.453</jenkins.version>
```

Comment:
could you update this to corresponding LTS line instead please?

Reply:
https://github.com/jenkinsci/dark-theme-plugin/pull/484 set the jenkins version to `2.453` so I assume there was a reason for that?
I could bump the baseline and jenkins version to to `2.462.x` instead. WDYT?

Reply:
yes please bump